src/sdk/model/json/cloud-tenant.ts

```diff
@@ -0,0 +1,61 @@
+import { EntityJson } from './entity';
+
+/**
+ * Interface for Cloud Tenant JSON properties.
+ */
+export interface CloudTenantJson extends EntityJson {
+  uid: string;
+  enabled: boolean;
+  resources: VeeamResource;
+  last_result: string;
+  last_active: number;
+  throttling_enabled: boolean;
+  throttling_speed_limit: number;
+  throttling_speed_unit: string;
+  public_ip_count: number;
+  crm: string;
+  owner_name: string;
+  contract_uuid: string;
+  location_id: string;
+  endpoint: string;
+  backup_count: number;
+}
+
+/**
+ * Interface for VeeamResource JSON properties.
+ */
+export interface VeeamResource {
```

Comment:
all of these interfaces that describe JSON formats should have names that end in `Json`